### PR TITLE
96 client meta infos

### LINKED ISSUE #96: Add logo field for the client
We need a string field to store base64 encoded image.

### PR DESCRIPTION
**Pull request by harti2006**
fixes #96

### CODE REVIEW

**Comment by harti2006** [comment]
this still needs to be tested properly

File: src/main/java/org/zalando/planb/provider/AuthorizeController.java [hunk 1]
```diff
@@ -109,7 +111,7 @@ String showAuthorizationForm(@RequestParam(value = PARAM_REALM) final Optional<S
                                  @RequestParam(value = PARAM_ERROR) final Optional<String> error,
                                  @RequestHeader(value = HEADER_HOST) final Optional<String> hostHeader, final Model model) {
 
-        checkReponseType(responseType);
+        checkResponseType(responseType);
```
Comment (hjacobs) [inline src/main/java/org/zalando/planb/provider/AuthorizeController.java]: good catch, I did not see that typo

**Comment by vroldanbet** [comment]
looks great :)

**Comment by hjacobs** [comment]
:+1:

**Pull request merged**